packages/storage/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@fuels/local-storage",
```

Comment:
Here also `@fuels/react-storage` we can add session-storage and other types inside this package on the future.